lwallet/Command.hs

```diff
@@ -111,6 +113,20 @@ switch = lexeme $ positive $> True <|>
     positive = text "+" <|> text "y" <|> text "yes"
     negative = text "-" <|> text "n" <|> text "no"
 
+stakeholderId :: Parser StakeholderId
+stakeholderId = lexeme $ do
+    str <- many1 alphaNum
+    case decodeTextAddress (toText str) of
```

Comment:
`StakeholderId` is `addressHash pk`. The easiest solution is to list not only addresses and PKs, but also `StakeholderId`s. Or you can add another command for it.